### ISSUE TITLE
Category image does not appear in listig grid item - JET ENGINE

### ISSUE THREAD

**Issue by MPHood** [Crocoblock/suggestions#384]
I created a metafield ( Media / image) with jet engine for post categories.
I assigned an image to every category.

I created a post listing that should display this category image (individually) for each post in the listing grid.

The single listing should look like this: Featured image  / title/ Description / **individual category image** / Read More

Every thing works fine, except of the (regular post) category image
It does not work neither as regular image, nor as backroundimage - i tried many variations.

maybe you can help me.

**Comment by Floydtm** [comment]
What type of source did you choose? For Categories and tags, you need to select [Terms](http://prntscr.com/ojfk0c). Perhaps the problem is this.

**Comment by MPHood** [comment]
I used Posts.

I need the listig grid to appear not only on category / archive pages -> In my case i want it on the frontapge (ist terms as a source right for that?).  

I just tried terms - but if i do that the dynamic title and the featured image does not work any longer in the listing grid - it shows same post (image ,Title..) over and over again (being in an categoryry archive, with at least 4 different posts).

so switching to terms did not realy help me
